### DISCUSSION TITLE
Home Assistant OS 12.2

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3303] [Announcements]
## Home Assistant Operating System

* Update Buildroot to 2024.02 (#3241) @sairon
* kernel: enable MPTCP support (#3248) @matttbe
* Bump buildroot to update Docker to v25.0.5 (#3271) @sairon
* Use Docker 25 for installing containers in hassio package (#3286) @sairon
* Only run HA CLI interactively if stdout is a terminal (#3292) @sairon
* Ignore non-existing CP15 barrier instructions emulation (#3285) @agners

## Raspberry Pi

* Disable UAS via usb-storage.quirks on RPi for ADATA SD600Q (#3281) @agglrx

## Home Assistant Green

* Linux: Update kernel to 6.6.25 (#3291) @sairon

## Open Virtual Appliance

* Linux: Update kernel to 6.6.25 (#3291) @sairon

## Generic x86-64

* Linux: Update kernel to 6.6.25 (#3291) @sairon

## Hardkernel ODROID

* Linux: Update kernel to 6.6.25 (#3291) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.25 (#3291) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.25 (#3291) @sairon

## Generic aarch64

* Linux: Update kernel to 6.6.25 (#3291) @sairon

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/12.2'>Home Assistant OS 12.2</a>.</em>

**Comment by TrilIiard** [comment]
RPI4 didnt restart while updating, instead of restarting it turned off and wont come back pressing the Power Button. Had to unplug the power cable, wait for 10 seconds, plug back in and it booted with 12.2

**Reply by agners** [reply]
Are you booting from SSD?

**Reply by TrilIiard** [reply]
Yes

**Comment by dtremere** [comment]
I have been using HAOS (on Lenovo M910q) for a few months now and normally don't have trouble with updates. I was able to do the HAOS update to 12.1 (back in Feb I think), but I have run the update for 12.2 6 or 8 times now and after rebooting my server and restarting, it just ends up back at 12.1 every time.
I am unsure where to look to find out what is failing during the update, can you point me in the right direction?
(I do have a keyboard/mouse/monitor on the server if that helps)
Thanks and sorry if this is silly question.
-Doug

**Reply by sairon** [reply]
Can you, shortly after it's reverted back to 12.1, check logs from the previous boot using `ha host logs -b-1 -n1000`? Hopefully something will be there.

**Comment by dtremere** [comment]
Hi Jan,
I am able to execute that command and see the log list on the screen of the
server, but I don't really know what I am looking for inside the log. Is
there a command that will let me send that log to a file or something, so I
can look through it more easily or even share it for assistance?
Sorry, that seems like something I should already know, I tried to look at
the help for commands but didn't see anything obvious.
Thanks so much,
Doug

**Comment by dtremere** [comment]
Scratch that!!
Sorry, I just tried running the update again and it worked this time.
I didn't do anything different than previously, just clicked on it and
basically ignored it (because I was sure it would fail - LOL).
Sorry about that, and thanks for your time.
-Doug

**Comment by jeroenterheerdt** [comment]
this update killed my HA install. It failed with this in the console: ```cifs vfs cifs_read_super get root inode failed``` . I am running HA on a VM in proxmox.  I had to restore a backup with OS 12.1.
